semgrep-core/src/parsing/Parse_pattern.ml

```diff
@@ -14,6 +14,8 @@
  *)
 open Common
 
+let logger = Logging.get_logger [ __MODULE__ ]
```

Comment:
Did you mean to add this debug logging here? I'm fine with it, but I want to make sure it was intentional.

Reply:
yep, it's used below to know if the pfff or tree-sitter pattern parser is used.
(I needed the information when I fixed this issue, and I do similar logging in Parse_target.ml)